oauthenticator/cilogon.py

```diff
@@ -160,24 +162,26 @@ def authenticate(self, handler, data=None):
         username = resp_json.get(self.username_claim)
         if not username:
             self.log.error("Username claim %s not found in the response: %s",
-                self.username_claim, sorted(resp_json.keys())
-            )
+                           self.username_claim, sorted(resp_json.keys())
+                           )
             raise web.HTTPError(500, "Failed to get username from CILogon")
 
         if self.idp_whitelist:
             gotten_name, gotten_idp = username.split('@')
             if gotten_idp not in self.idp_whitelist:
-                self.log.error("Trying to login from not whitelisted domain %s", gotten_idp)
-                raise web.HTTPError(500, "Trying to login from not whitelisted domain")
+                self.log.error(
+                    "Trying to login from not whitelisted domain %s", gotten_idp)
+                raise web.HTTPError(
+                    500, "Trying to login from not whitelisted domain")
             if len(self.idp_whitelist) == 1 and self.strip_idp_domain:
                 username = gotten_name
         userdict = {"name": username}
         # Now we set up auth_state
         userdict["auth_state"] = auth_state = {}
-        # Save the access token and full CILogon reply in auth state
+        # Save the token response and full CILogon reply in auth state
         # These can be used for user provisioning
         #  in the Lab/Notebook environment.
-        auth_state['access_token'] = access_token
+        auth_state['token_response'] = token_response
```

Comment:
for backward-compatibility maybe leave access_token there?

Otherwise +1

Reply:
Done.  Although I have a suspicion that I may be the only current user.

Reply:
entirely possible, but still nice to do it anyway